### ISSUE TITLE
Restore an unsaved file OR Avoid accidentally typing `:wq!` as `q!`

### ISSUE THREAD

**Issue by FYimu** [nus-cs2030/2122-s1#67]
I accidentally typed `:wq!` as `q!` while writing Simulator.java for the project during recess week (I'm used to override because I always have the error `E510: Can't make backup file`), and I couldn't find a way to restore my unsaved file with the methods available online. I post the problem here just in case someone else also experiences the same problem and looks for a solution :>

On a side note, using `ZZ` (note the capital letter) has the same effect as `:wq`, which might potentially be more convenient and safer XD

**Comment by RussellDash332** [comment]
It's one of the instances where it is barely possible to fix the damage. Your suggestion to just `ZZ` is viable and helps others as well! 👍🏻 

For myself, I always use `wa!` to save everything on my editor because I will (almost) always open multiple Java files at once and go back to the terminal by `Ctrl+Z` and back again by `fg`. Once I'm done, I will use `wqa!` to close all of them.

My problem was not `wq!` VS `q!` though, I accidentally `scp`-ed one of my files to another file so that the latter's content is now overwritten by the former. The only way to resolve it in my head is to simply....

<details>
<summary>
Spoiler.
</summary>
retype it.
</details>

**Comment by RussellDash332** [comment]
> @RussellDash332 big lollll now the perks of not knowing so many commands 🤣 Thanks for the useful tips!!! Gonna try them out next time 👍

Yeah every single lab we just list all the class names then for every class name just append the Java file behind `vim -p`
E.g. `vim -p Fruit.java Orange.java Apple.java`

Navigate through each file/tab by `gt` or `gT`. `a` in `wa` or `wqa` means 'all'.

**Comment by Gibson0918** [comment]
You can use :tabnew xxx.java to open open while in vim

**Comment by BrendanCheong** [comment]
> > @RussellDash332 big lollll now the perks of not knowing so many commands 🤣 Thanks for the useful tips!!! Gonna try them out next time 👍
> 
> Yeah every single lab we just list all the class names then for every class name just append the Java file behind `vim -p` E.g. `vim -p Fruit.java Orange.java Apple.java`
> 
> Navigate through each file/tab by `gt` or `gT`. `a` in `wa` or `wqa` means 'all'.

wow thats awesome
